futures-util/src/stream/mod.rs

```diff
@@ -740,6 +740,30 @@ pub trait StreamExt: Stream {
     ///
     /// This method is only available when the `std` feature of this
     /// library is activated, and it is activated by default.
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// #![feature(async_await, await_macro)]
+    /// # futures::executor::block_on(async {
+    /// use futures::prelude::*;
```

Comment:
We're removing the prelude from the doc examples. That makes it clearer which traits are involved:

```rust
use futures::stream::{self, StreamExt};
```

Reply:
Done